pkg/term/term_windows.go

```diff
@@ -48,8 +48,8 @@ func GetWinsize(fd uintptr) (*Winsize, error) {
 
 	// TODO(azlinux): Set the pixel width / height of the console (currently unused by any caller)
 	return &Winsize{
-		Width:  uint16(info.Window.Bottom - info.Window.Top + 1),
-		Height: uint16(info.Window.Right - info.Window.Left + 1),
+		Width:  uint16(info.Window.Right - info.Window.Left + 1),
+		Height: uint16(info.Window.Bottom - info.Window.Top + 1),
```

Comment:
I think those `+ 1` are wrong: if I run an interactive session and sleep on the `a` key, it wraps one character "too late", and gives me empty lines. Behavior is correct with the `+1`.